portfolio/src/main/webapp/script.js

```diff
@@ -11,51 +11,53 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-$(document).ready(function() {
+$(document).ready(() =>  {
   getComments();
 
-  $('.deleter').click(function(event) {
-    event.preventDefault();
-    $.ajax({
-      url: '/delete-data',
-      type: 'POST',
-      success: function(response) {
-        $('.comments-section').empty();
-      }
-    });
+  $('#deleter').click((event) => {
+    deleteComments(event);
   });
 
-  $('a').click(function() {
+  $('a').click(() => {
     animateScrollTop($(this).attr('href').substr(1));
   });
 
-  $('.contact-form').submit(function(event) {
+  $('.contact-form').submit((event) => {
     event.preventDefault();
     const name = $('.form-name').val();
     const comment = $('.form-comment').val();
     postComment(comment, name);
   });
 
-  $('.comments-editor').submit(function(event) {
+  $('#comments-editor').submit((event) => {
     event.preventDefault();
     $.ajax({
       url: '/comments',
       type: 'GET',
       data: '&num=' + $('.comments-num').val(),
       success: function(response) {
         const jsonComments = JSON.parse(response);
-        const comments = jsonComments['comments'];
+        const comments = jsonComments.comments;
         $('.comments-section').empty();
-        for (let index = 0; index < comments.length; index++) {
-          const comment = comments[index];
-          $('.comments-section').append(`<p>${comment.comment}</p>`);
-          $('.comments-section').append(`<p>${comment.name}</p>`);
+        for (const comment of comments) {
+          addCommentToDOM('.comments-section', comment);
         }
       }
     });
   });
 });
 
+function deleteComments(event) {
+  event.preventDefault();
+  $.ajax({
+    url: '/delete-data',
+    type: 'POST',
+    success: function(response) {
```

Comment:
nit: you could use an arrow function here too.

Reply:
done!